pages/vaults/index.tsx

```diff
@@ -216,14 +216,13 @@ function Index(): ReactElement {
 	**	implemented as a simple string.includes() on the vault name.
 	**********************************************************************************************/
 	const searchedVaultsToDisplay = useMemo((): TYDaemonVault[] => {
-		const vaultsToUse = [...vaultsToDisplay];
-
```

Comment:
This seemed unnecessary, spreading the vaults to just put them back again into an array